# Layout of one dimension of acquisition of a multilayer laser scan.
# From this definition, you can compute the exact angles and timestamps of
# individual rays or subscans.

AngularOffsets angular_offsets
TimeOffsets time_offsets